build/dockerfiles/Dockerfile

```diff
@@ -19,6 +19,13 @@ RUN ./index.sh > /build/devfiles/index.json
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
 RUN chmod -R g+rwX /build/devfiles
 
+FROM docker.io/node:12.20.1-alpine3.12 as dwtemplates
```

Comment:
should we use a recent LTS version there ? like nodejs 16

Reply:
Changed to `node:lts-alpine3.14`